package.json

```diff
@@ -17,6 +18,12 @@
   },
   "homepage": "https://github.com/MartynasLun/CypressWorkshop#readme",
   "dependencies": {
-    "cypress": "^5.0.0"
+    "cypress": "^5.0.0",
+    "fs-extra": "^9.0.1"
+  },
+  "devDependencies": {
+    "mochawesome": "^6.1.1",
+    "mochawesome-merge": "^4.1.0",
+    "mochawesome-report-generator": "^5.1.0"
```

Comment:
Why some of them are "dependencies" and others "devDependencies"? :) I would suggest to read a little bit about the differences.

Reply:
No particular reason. Moved `devDependencies` to `dependencies` as the project only contains test code.